client/packages/coldchain/src/Equipment/DetailView/StatusForm.tsx

```diff
@@ -140,7 +174,39 @@ export const StatusTab = ({ draft, onChange, value }: AssetLogPanel) => {
             onChange={e => debouncedOnChange({ comment: e.target.value })}
           />
         </Row>
+        <Box sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          marginTop: 2,
+          padding: 0,
+          alignItems: 'center',
+          height: '100%',
+          width: '100%',
+          justifyContent: 'center',
+        }}>
+          {!isSmallScreen && <Upload onUpload={onUpload} />}
+          {isSmallScreen && isNative && (
+            <>
+              <UploadButton onUpload={onUpload} files={draft.files} customLabel={t('button.browse-files')}></UploadButton>
+              <TakePhotoButton onUpload={onUpload} files={draft.files} />
+            </>
+          )}
+          {isSmallScreen && !isNative && (
+            <>
+              <UploadButton onUpload={onUpload} files={draft.files} customLabel={t('button.browse-files')}></UploadButton>
+              <UploadButton onUpload={onUpload} files={draft.files} customLabel={t('button.camera')}></UploadButton>
+            </>
+          )}
```

Comment:
I think it might be clearer if we abstracted these buttons out, I'll make a commit to show you what I mean.